cmd/buildkitd/config/gcpolicy_windows.go

```diff
@@ -3,10 +3,36 @@
 
 package config
 
+import "golang.org/x/sys/windows"
+
 func DetectDefaultGCCap() DiskSpace {
-	return DiskSpace{Bytes: defaultCap}
+	return DiskSpace{Percentage: 10}
 }
 
 func (d DiskSpace) AsBytes(root string) int64 {
-	return d.Bytes
+	if d.Bytes != 0 {
+		return d.Bytes
+	}
+	if d.Percentage == 0 {
+		return 0
+	}
```

Comment:
While we're here, I think we should refactor the `DiskSpace.AsBytes` method into `gcpolicy.go`, and then just have a simple helper `getDiskSpace` that we can in `gcpolicy_unix.go`/`gcpolicy_windows.go`. No need to duplicate all the rest of the logic, and that should help improve readability here.